{"name": "Попытка входа с невалидными данными: неверный пароль", "status": "passed", "steps": [{"name": "Подготовка тестовых данных для сценария: неверный пароль", "status": "passed", "start": 1758210123832, "stop": 1758210123832}, {"name": "Отправляем POST-запрос для авторизации курьера", "status": "passed", "start": 1758210123832, "stop": 1758210124318}, {"name": "Проверяем код ответа 404", "status": "passed", "start": 1758210124318, "stop": 1758210124318}, {"name": "Проверяем сообщение об ошибке", "status": "passed", "start": 1758210124318, "stop": 1758210124318}], "parameters": [{"name": "test_case", "value": "'неверный пароль'"}, {"name": "credentials", "value": "<function TestCourierLogin.<lambda> at 0x0000025C0D73CD60>"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1758210123832, "stop": 1758210124320, "uuid": "e6afd130-fc75-467f-8e0c-72f129620390", "historyId": "7438b60096b91fabc6b4a581b8e6505a", "testCaseId": "505bba1c9113ea857d47e178537c8295", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_negative_cases", "labels": [{"name": "story", "value": "Неуспешная авторизация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "What-Is-Lowe"}, {"name": "thread", "value": "5532-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}